lib/resources/yum.rb

```diff
@@ -22,6 +20,7 @@
 # describe yum.repo('epel') do
 #   it { should exist }
 #   it { should be_enabled }
+#   its('baseurl') { should include 'mycompany.biz' }
```

Comment:
The key in the yum output and in the `*.repo` files is `baseurl` so I vote for keeping it consistent.